doc/05-service-monitoring.md

```diff
@@ -902,6 +902,15 @@ Instead, choose a plugin and configure its parameters and thresholds. The follow
 
 ### Windows Monitoring <a id="service-monitoring-windows"></a>
 
+!!! important
+
+    [Icinga for Windows](https://icinga.com/docs/icinga-for-windows/latest/doc/000-Introduction/)
+    is the recommended way to monitor Windows via Icinga 2.
+    Even if the plugins it ships out-of-the-box don't already cover your needs, you can
+    [create your own](https://icinga.com/docs/icinga-for-windows/latest/doc/900-Developer-Guide/11-Custom-Plugins/).
+
+Other probably still working solutions include:
```

Comment:
"probably still working" sounds quite indecisive. Maybe more like "The following alternative solutions exist but are no longer recommended for new installations:".

Reply:
Sure. Of course, that's my intention! When have you tested which of them the last time?

Reply:
I did not make a statement about the state of these options. But if you have information that some of these are broken, we shouldn't list them.

Looking at the [surrounding section in the docs](https://icinga.com/docs/icinga-2/latest/doc/05-service-monitoring/#service-monitoring-overview), it just provides a list of options and a big info box might even look out of place. I think it could be as simple as a new entry at the top of the list "Recommended: Check Plugins provided by Icinga for Windows" (with corresponding links). The goal is to mention IfW and have a hint that this is the preferred options, I think that suggestion would achieve that and fit the context.